engine/browserParser.js

```diff
@@ -1,18 +1,20 @@
 import cache from './cache.js';
 
 export default class BrowserParser {
-  results = {
```

Comment:
Wouldn't this only work if we added stage-1?

The linter fails without this change:
```
[15:39:18] Starting 'lint'...
[15:39:18] 'lint' errored after 337 ms
[15:39:18] ESLintError in plugin 'gulp-eslint'
Message:
    Parsing error: Unexpected token =
Details:
    fileName: engine/browserParser.js
    lineNumber: 4
[15:39:18] 
engine/browserParser.js
  4:12  error  Parsing error: Unexpected token =

✖ 1 problem (1 error, 0 warnings)
```